examples/simple-kafka-cluster.yaml

```diff
@@ -4,14 +4,15 @@ kind: ZookeeperCluster
 metadata:
   name: simple-zk
 spec:
+  version: 3.5.8
   servers:
     roleGroups:
       default:
+        selector:
+          matchLabels:
+            kubernetes.io/os: linux
         replicas: 3
-        config:
-          myidOffset: 10
-  version: 3.5.8
-  stopped: false
+        config: {}
```

Comment:
Shouldn't need `config:` at all then?

Reply:
No but i like to keep it in the example in case anyone wants to set something (after reading the docs carefully!)